vulture/config.py

```diff
@@ -158,6 +159,12 @@ def csv(exclude):
         default=missing,
         help="Sort unused functions and classes by their lines of code.",
     )
+    parser.add_argument(
+        "-c", "--config",
```

Comment:
```suggestion
        "--config",
```
Let's be explicit here. This is also in line with ruff and black.